src/main/dotnet/Program.cs

```diff
@@ -1,104 +0,0 @@
-﻿using Microsoft.Build.BuildEngine;
-using Newtonsoft.Json.Linq;
-using System;
-using System.IO;
-using System.Linq;
-
-namespace ProjectFileParser
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            try
-            {
-                Parse(args[0]);
-            }
-            catch (Exception e)
-            {
-                Console.Error.WriteLine("Error during project file parsing: {0}", e);
-                Console.Error.WriteLine(e.StackTrace);
-                Environment.ExitCode = -1;
-            }
-        }
-
-        static void Parse(String file)
-        {
-            var obj = JObject.Parse(Console.In.ReadToEnd());
```

Comment:
See my implementation [there](https://github.com/Ullink/gradle-msbuild-plugin/commit/a1bc3a4209fecdcf6dd63b082c0e269b45d5f4d8) feel free to prefer yours :)